src/ihela_client/banking_client.py

```
@@ -0,0 +1,412 @@
+                "Authorization": "{} {}".format(
+                    self.auth_token_object["token_type"],
+                    self.auth_token_object["access_token"],
+                )
+            }
+        return {}
+
+    def authenticate(self):
+        url = iHela_TOKEN_URL
+        auth_data = {"grant_type": "client_credentials"}
+        try:
+            resp = requests.post(
```

Ruff (0.15.21)

[error] Probable use of `requests` call without timeout

(S113)